### ISSUE TITLE
Please Add Auto Clean Feature

### ISSUE THREAD

**Issue by Poveretto** [DeweyReed/ClipboardCleaner#19]
1.Normal mode -  be cycled to clean clipboard (user-custom time span)
2.Blacklist mode - intergrated with KEYWORDFILTER
       (2.1 clean clipboard once  apps in blacklist  access to clipboard --no KEYWORD
        2.2  When use apps in blacklist, clean specific item if KEYWORDFILTER is on)

**Comment by DeweyReed** [comment]
I don't think we could know which apps are accessing the clipboard. Thus blacklist mode is impossible.
